src/content/docs/devops.mdx

```diff
@@ -511,6 +546,8 @@ pnpm prebuild && pnpm build:curriculum && pnpm build:server
 pnpm start:server
 ```
 
+</Steps>
```

Comment:
Step 9 above, has double space `9.  Start Instances` after the dot, causing error